drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -5,27 +5,33 @@
         width:{{ width_percent }}%;
         height:{{ height_percent }}%;">
         <p>{{{ title }}}</p>
+        <p class="sr">{{{ description }}}</p>
     </div>
 </script>
 
 <script id="zone-input-tpl" type="text/html">
     <div class="zone-row {{ id }}" data-index="{{index}}">
-       <label>{{i18n "Text"}}</label>
-       <input type="text" class="title" value="{{ title }}" />
-       <a href="#" class="remove-zone hidden">
-           <div class="icon remove"></div>
-       </a>
-       <div class="layout">
-           <label>{{i18n "width"}}</label>
-           <input type="text" class="size width" value="{{ width }}" />
-           <label>{{i18n "height"}}</label>
-           <input type="text" class="size height" value="{{ height }}" />
-           <br />
-           <label>x</label>
-           <input type="text" class="coord x" value="{{ x }}" />
-           <label>y</label>
-           <input type="text" class="coord y" value="{{ y }}" />
-       </div>
+        <label>{{i18n "Text"}}</label>
```

Comment:
The `label` elements in this section are not linked to the `input` elements (by wrapping them or with `for` attributes)